src/Link.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Text from './Text'
+
+const anchorText = Text.withComponent('a')
```

Comment:
Since the `<a>` tag generally inherits all the typographic styles of its parent, I don't think this needs to extend the `Text` component – i.e. I think this PR should be as simple and as small a change as possible

Reply:
Good point about `<a>` tag inherits parent's styles. Rewrote to simply the `Link` to just a simple styled anchor with text decoration none.

My reasoning is that most of the designs seem to use links without any underlines or any other type of text decoration. If using of the `Link` wants a different text decorator, the user can style it via the containing element and let the style be inherited by `Link`.